src/main/resources/config.properties

```diff
@@ -4,8 +4,8 @@ state = Baden-Württemberg
 # Comma separated list of locations. Optional, if you already have a placement code just add it in square brackets after the place. Furthermore you can optionally specify the related server code next to the placement code in parentheses (can be found in the URL e.g. "001-iz.impfterminservice.de -> server code == 001).
 locations = 69124 Heidelberg[XXXX-XXXX-XXXX](XXX),76137 Karlsruhe
 
-# Your age. Used in age verification field.
-personAge = 42
+# Your age. Used in birthdate verification field. DD.MM.YYYY
+personBirthdate = 31.12.1994
```

Comment:
You also have to change name and type in the `Config` class (`kotlin/de/tfr/impf/config/Config.kt`):

```kotlin
val personAge: Int by lazyIntProperty()
// replace by
val personBirthdate : String by lazyProperty()
```
In addition, you should also replace the `personAge` in the `README`.